lib/index.js

```diff
@@ -17,8 +17,17 @@ const routes = bedrock.config['session-http'].routes;
 // add routes
 bedrock.events.on('bedrock-express.configure.routes', function(app) {
   app.get(routes.session, asyncHandler(async (req, res) => {
-    // emit event with request and session and let listeners add session data
+    // the originalMaxAge is in ms and will result
+    // in an accurate expiration time even if the client's
+    // time is incorrect
     const session = {};
+    const {originalMaxAge} = req.session.cookie || {};
+    if(originalMaxAge) {
+      session.originalMaxAge = originalMaxAge;
+    }
+    // return the ttl with each refresh
+    session.ttl = bedrock.config.express.session.ttl;
```

Comment:
Note that this will have to be added to `bedrock-express` as a config var before this will work.

Reply:
it was but I need to test this before I can confirm this all works: https://github.com/digitalbazaar/bedrock-express/commit/6aa0147e082a53fcb801cd59281bc2a6b68bf633

Reply:
The ttl is in the session and is working: https://github.com/digitalbazaar/bedrock-session-http/pull/2/commits/9cdd55328efa2279a526a5c917e9fce1f153656b